src/axes/categoryAxis.ts

```diff
@@ -300,6 +300,12 @@ namespace Plottable.Axes {
       let xTranslate = this.orientation() === "right" ? this._tickSpaceRequired() : 0;
       let yTranslate = this.orientation() === "bottom" ? this._tickSpaceRequired() : 0;
       Utils.DOM.translate(this._tickLabelContainer, xTranslate, yTranslate);
+
+      // hide ticks and labels that overflow the axis
+      this._showAllTickMarks();
+      this._showAllTickLabels();
+      this._hideOverflowingTickLabels();
+      this._hideTickMarksWithoutLabel();
```

Comment:
Sometimes if the space for a tick label is too small, it won't get drawn: https://jsfiddle.net/hellochar/t8j5d7vt/ Will these "non-labelled tick-marks" get removed now?